src/main/java/com/googleinterns/zoomtube/utils/TranscriptLineUtil.java

```diff
@@ -18,17 +18,15 @@
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
 import com.googleinterns.zoomtube.data.TranscriptLine;
-import java.util.Date;
-import java.util.concurrent.TimeUnit;
 
 /** Provides methods to create TranscriptLine Entities and TranscriptLine objects. */
 public final class TranscriptLineUtil {
   public static final String KIND = "TranscriptLine";
   public static final String LECTURE = "lecture";
-  public static final String START = "start";
-  public static final String DURATION = "duration";
+  public static final String START_TIMESTAMP_MILLISECONDS = "start_milliseconds";
+  public static final String DURATION_MILLISECONDS = "duration_milliseconds";
```

Comment:
These strings ought to be able to be left as "start" and "duration" there should not be any other string of its kind stored in the db.